frontend/pages/layouts/index.js

```diff
@@ -16,15 +42,21 @@ export default () => {
           Add +
         </Link>
       </div>
-      {layouts.map((layout, i) => 
-        <div key={i} className='mt-4'>
-          <LayoutCard
-            layout={layout}
-          />
-        </div>
-      )}
+      <InfiniteScroll
+        pageStart={1}
+        hasMore={hasMore}
+        loadMore={loadMoreLayouts}
+      >
+        {layouts.map((layout, i) =>
+            <div key={i} className='mt-4'>
+              <LayoutCard
+                  layout={layout}
+              />
+            </div>
+        )}
+      </InfiniteScroll>
     </div>
   )
 }
 
-export const action = ({ dispatch }) => dispatch(loadLayouts())
+export const action = ({ dispatch }) => dispatch(loadLayouts(0, pageSize))
```

Comment:
use defaults. It doesn't make a difference but just looks cleaner

Reply:
do you mean like this? `export const action = ({ dispatch }) => dispatch(loadLayouts({}))` and then it will take up default values.